lib/index.js

```diff
@@ -4,6 +4,32 @@
 var os = require('os');
 var _ = require('underscore');
 /**
+ * Default options
+ */
+var options;
+var defaultOptions = {
```

Comment:
no but I also think options shouldn't be scoped to the whole module either, was just trying to return it to the previous working state. I'll revert.